lib/vbox/machine.go

```diff
@@ -52,11 +50,9 @@ func CheckMachine(machine, device string) error {
 	fmt.Println("[+] Checking virtual machine")
 	// checking file location
 	if !fileExists(machinePath) {
-		if device == constants.DEVICE_TYPE_EDISON {
-			repository = repo.VirtualBoxRepositoryEdison()
-		} else {
-			repository = repo.VirtualBoxRepository()
-		}
+
+		var err error
+		repository, err = repo.NewRepositoryVM()
```

Comment:
__[gometalinter]__
* [ineffassign] ineffectual assignment to err

<!-- Comment by [SideCI](https://sideci.com) -->